pkg/ccl/changefeedccl/kvfeed/kv_feed.go

```diff
@@ -216,8 +216,13 @@ func (f *kvFeed) run(ctx context.Context) (err error) {
 				}
 			}
 		}
+
 		// Exit if the policy says we should.
 		if boundaryType == jobspb.ResolvedSpan_RESTART || boundaryType == jobspb.ResolvedSpan_EXIT {
+			// Close the sink so it knows no more writes are expected
+			if err := f.sink.Close(ctx); err != nil {
+				log.Warningf(ctx, "failed to close buffer: %s", err)
```

Comment:
s/bufffer/sink/?

Reply:
I think I went with buffer in the log message so it wouldn't be confused with the other user of the word `sink`.